test/src/test/java/org/corfudb/integration/StateTransferIT.java

```diff
@@ -281,9 +282,14 @@ private void verifyStateTransferWithNodeRestart(int restartNode) throws Exceptio
         final int timeOut = 200;
         return CompletableFuture.runAsync(() -> {
             while (stopFlag.get()) {
-                TokenResponse token = writerRuntime.getSequencerView().next();
-                writerRuntime.getAddressSpaceView().write(token, "Test Payload".getBytes());
-                Sleep.sleepUninterruptibly(Duration.ofMillis(timeOut));
+                try {
+                    TokenResponse token = writerRuntime.getSequencerView().next();
+                    writerRuntime.getAddressSpaceView().write(token, "Test Payload".getBytes());
+                    Sleep.sleepUninterruptibly(Duration.ofMillis(timeOut));
+                }
+                catch (OverwriteException oe) {
+                    // Continue writing
```

Comment:
Looks like, this exception shouldn't appear, looks like we broke streamlog recently

Reply:
If this exception is not expected and this test ignores it, then why approve?

Reply:
Also, how recent is "recently" ? This bug was reported back in august #2740